### PR TITLE
StepOut breakpoint fix

### LINKED ISSUE #1164: [Debugger bug] Step out doesn't work unless preceded by step in/over
**PCSX2 version:**
v1.5.0-dev-269-gd59e4ca (2016-02-01)

**PCSX2 options:**
Defaults. No options modifications affect this issue.

**Plugins used:**
Defaults (GSdx32-AVX, LyliPad, SPU2-X, cdvdGigaherz, USBnull, FWnull and DEV9ghzdrk). No particular plugins affect this issue.

**Plugin settings:**
GSdx (Direct3D11 software mode). No settings modifications affect this issue.

**Description:**
In PCSX2's debugger, pressing the "Step Out" button right after reaching a breakpoint doesn't currently do anything. It only begins to work if the "Step In" or "Step Over" buttons are pressed beforehand.

**How to replicate:**

1. Launch PCSX2 and fast boot any game.
2. Open the debugger and add a breakpoint that you're sure will be reached (has to be deeper in the game's ELF logic than the entrypoint).
3. Fast boot the game again and wait for the breakpoint to be triggered.
4. Press the "Step Out" button repeatedly. Nothing will happen, apart from some parts of the debug window flickering and "EE/iR5900-32 Recompiler Reset" getting added to the log window.
5. Press the "Step In" and/or "Step Over" button. They'll work as expected.
6. Press the "Step Out" button once more. This time, it'll work as it's supposed to (come out of the function the breakpoint was in).
7. Step 4 didn't work as expected. It should've been able to produce the same result as step 6. Step 6 only worked because step in/out were used beforehand.

**Last known version to work:**
Never worked. Oldest tested version: pcsx2-v1.3.1-187-g0ab0c6c-windows-x86 (2015-04-27).

**PC specifications:**
CPU: Intel Core i7 2600K @ 3.4GHz; GPU: XFX Radeon HD 6970 2GB; OS: Windows 7 SP1

**Other comments:**
@Kingcom FYI :).

### PR DESCRIPTION
**Pull request by NZJenkins**
Fixes #1164

### CODE REVIEW

**Comment by gregory38** [comment]
A rebase will better than a merge. But it seems fine for me

**Comment by turtleli** [comment]
Seems to work here. I'll merge once it's rebased.

File: pcsx2/gui/Debugger/DisassemblyDialog.cpp [hunk 1]
```diff
@@ -458,6 +458,9 @@ void DisassemblyDialog::stepOut()
 {
 	if (!r5900Debug.isAlive() || !r5900Debug.isCpuPaused() || currentCpu == NULL)
 		return;
+
```
Comment (turtleli) [inline pcsx2/gui/Debugger/DisassemblyDialog.cpp]: Oh, could you remove the whitespace here as well.

**Comment by ssakash** [comment]
Hmm, that's weird. normally the trailing white space should have been removed if you had rebased to master without you needing to manually remove the trailing white space. (due to editorconfig)

did you remove the trailing whitespace before rebasing ? if yes, that makes sense.

**Comment by gregory38** [comment]
I don't think Editorconfig apply to git (github and some editors)

**Comment by turtleli** [comment]
When I said trailing whitespace I meant the tab character on the blank line, not the blank line itself, but I didn't write trailing in the initial message (it was later edited), so that's probably my fault. So let's merge.

**Comment by Aced14** [comment]
@ssakash EditorConfig support is built into a few editors, but is missing from many others. Standalone plugins exist for the others. Check out the [plugins section on EditorConfig's website](http://editorconfig.org/#download) for more information.

It's not built into Git in the same manner as .gitattributes/.gitignore are.

**Pull request merged**